algorithm/cloudflare/redis_store.go

```diff
@@ -11,7 +11,8 @@ import (
 )
 
 type RedisDataStore struct {
-	RDB algorithm.Rediser
+	RDB            algorithm.Rediser
+	ExpirationTime time.Duration
```

Comment:
Do we need it here? Cant' we have just `window * 2` in the `func (s *RedisDataStore) Inc(key string, window time.Time) error` function directly? or the `window` is not same as `period`?

Reply:
yes this is not a same, can't do it inside `Inc` function